src/util/date.ts

```diff
@@ -0,0 +1,5 @@
+const currentYear = new Date().getFullYear();
+const years = Array.from(new Array(currentYear - 1899), (val, index) => currentYear - index);
+const months = Array.from(Array(12), (val, index) => 1 + index);
```

Comment:
```suggestion
const years = Array.from(new Array(currentYear - 1899), (year, index) => currentYear - index);
const months = Array.from(Array(12), (month, index) => 1 + index);
```

val 대신 명확하게 작성할 수 있으면 좋을거 같아요 👍
`from` 은 가독성이 떨어질 수 있어서

```
const months = new Array(12).fill(null)
  .map((_, idx) => idx + 1)
```

하는 것도 방법일 것 같아 !